src/library/dao/playlistdao.h

```diff
@@ -59,6 +59,9 @@ class PlaylistDAO : public QObject, public virtual DAO {
     int createUniquePlaylist(QString* pName, const HiddenType type = PLHT_NOT_HIDDEN);
     // Delete a playlist
     void deletePlaylist(const int playlistId);
+    /// Delete Playlists with fewer entries then "length"
+    /// @return number of deleted playlists, -1 on error
+    int deleteAllPlaylistsWithFewerItems(PlaylistDAO::HiddenType type, int length);
```

Comment:
It doesn't hurt to let the parameters speak for themselves: "length" -> "minNumberOfTracks"

Each "item" in a playlist is a track, we do not need to introduce a generic term here: -> deleteAllPlaylistsWithFewerTracks